src/main/webapp/help.html

```diff
@@ -6,17 +6,18 @@
     <link rel="stylesheet" href="assets/css/custom_style.css">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Help for Sections Manager</title>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
```

Comment:
i have added https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js since the sections uses that version